src/Components/ArticleCard.jsx

```diff
@@ -7,11 +9,13 @@ const ArticleCard = ({ article }) => {
         alt={article.title}
       />
       <div className="article-block">
+      <Link to={`/articles/${article.article_id}`}>
         <h2 className="article-title">Article: {article.title}</h2>
+        </Link>
         <strong><p className="article-author">Written by: {article.author}</p>
```

Comment:
I would do this with CSS - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong